contrib/builder/rpm/amd64/generate.sh

```diff
@@ -111,15 +111,15 @@ for version in "${versions[@]}"; do
 	esac
 
 	case "$from" in
-		oraclelinux:6)
+		oraclelinux:6|amazonlinux:latest)
 			# doesn't use systemd, doesn't have a devel package for it
 			packages=( "${packages[@]/systemd-devel}" )
 			;;
 	esac
 
 	# opensuse & oraclelinx:6 do not have the right libseccomp libs
 	case "$from" in
-		opensuse:*|oraclelinux:6)
+		opensuse:*|oraclelinux:6|amazonlinux:latest)
```

Comment:
As far as I can see Amazon Linux does not ship `libseccomp` at all. I don't think disabling seccomp is acceptable for distributing Docker; we understand that some of the older distros have incompatible old versions but it is a highly recommended part of Docker security, and I am not happy about adding a new distro that does not support seccomp.